command/validate.py

```diff
@@ -1,8 +1,239 @@
+import logging
+import json
+
+logger = logging.getLogger(__name__)
+
 class ValidateCmd():
-    name = 'validate'
+    dataKey = "data"
```

Comment:
These are also used in the build method, could we put these values somewhere shared?

Reply:
IMO limiting cross dependencies is important in these modules --- and I'm not sure I see value to creating a "shared" module just for these strings. If they shared something more fundamental though and a shared module was required we could always refactor this to move these definitions.

Reply:
Cross dependencies should certainly be avoided - my  hesitation stems from a potential change to the bundle structure, which could come from outside teams, where we would need to update these keys in multiple areas (in this case the build, identify, and validate classes). This isn't a blocking issue, but something we may want to add as an enhancement.

Reply:
The suggestion of a bundle class sounds good to me.